packages/diagrams/backend/sirius-components-diagrams-layout/src/main/java/org/eclipse/sirius/components/diagrams/layout/services/LayoutStrategyEngineService.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.diagrams.layout.services;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import org.eclipse.sirius.components.diagrams.layout.incremental.ILayoutStrategyEngine;
+import org.eclipse.sirius.components.diagrams.layout.incremental.ILayoutStrategyEngineRegistry;
+import org.eclipse.sirius.components.diagrams.layout.incremental.data.NodeLayoutData;
+
+/**
+ * Service used to get the first {@link ILayoutStrategyEngine} available for the {@link NodeLayoutData}.
+ *
+ * @author gcoutable
+ */
+public class LayoutStrategyEngineService implements ILayoutStrategyEngineService {
+
+    private final ILayoutStrategyEngineRegistry registry;
```

Comment:
Ask for a `List<ILayoutStrategyEngine>` instead

Reply:
The service has been removed